README.md

```diff
@@ -146,6 +146,10 @@ The community maintained a list of **Algorithms and Data Structures** implementa
       <td><code>gr-dij</code></td>
       <td class="text-center">&#10004;</td>
       <td class="text-center">&#10004;</td>
+<<<<<<< HEAD
```

Comment:
please check the changes again. these parts are some remaining texts from conflicts. you have to remove them.